README.md

```diff
@@ -9,7 +9,7 @@
 
 ## Overview
 
-**Kyma** `/kee-ma/` is a platform for extending applications with microservices and [serverless](https://kyma-project.io/docs/components/serverless/#overview-overview) Functions. It provides CLI and UI through which you can connect your application to a Kubernetes cluster and expose the application's API or events securely thanks to the in-built [Application Connector](https://kyma-project.io/docs/components/application-connector/#overview-overview). You can then implement the business logic you require by creating microservices or serverless Functions, and triggering them to react to particular events or calls to your application's API. To limit the time spent on coding, use the in-built cloud services from the [Service Catalog](https://kyma-project.io/docs/master/components/service-catalog/#overview-service-catalog), exposed by [Service Brokers](https://kyma-project.io/docs/master/components/service-catalog/#overview-service-brokers) from such cloud providers as GCP, Azure, and AWS.
+**Kyma** `/kee-ma/` is a platform for extending applications with microservices and [serverless](https://kyma-project.io/docs/components/serverless/#overview-overview) Functions. It provides CLI and UI through which you can connect your application to a Kubernetes cluster and expose the application's API or events securely thanks to the built-in [Application Connector](https://kyma-project.io/docs/components/application-connector/#overview-overview). You can then implement the business logic you require by creating microservices or serverless Functions, and triggering them to react to particular events or calls to your application's API. To limit the time spent on coding, use the built-in cloud services from the [Service Catalog](https://kyma-project.io/docs/master/components/service-catalog/#overview-service-catalog), exposed by [Service Brokers](https://kyma-project.io/docs/master/components/service-catalog/#overview-service-brokers) from such cloud providers as GCP, Azure, and AWS.
```

Comment:
I decided to post it as a separate comment as I leave it to your discretion to apply it or not.

These sentences, are really long and not the easiest to read:
1) `It provides CLI and UI through which you can connect your application to a Kubernetes cluster and expose the application's API or events securely thanks to the built-in [Application Connector](https://kyma-project.io/docs/components/application-connector/#overview-overview).`
2) `You can then implement the business logic you require by creating microservices or serverless Functions, and triggering them to react to particular events or calls to your application's API.`

You could split them in smaller chunks. For example: 
1) `It provides CLI and UI through which you can connect your application to a Kubernetes cluster. You can also expose the application's API or events securely thanks to the built-in [Application Connector](https://kyma-project.io/docs/components/application-connector/#overview-overview).`
2) `You can then implement the business logic you require by creating microservices or serverless Functions. Trigger them to react to particular events or calls to your application's API.`

Reply:
Yup, I split them into two sections.